packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -178,25 +169,8 @@ export const Table = memo((props: Props) => {
       return;
     }
 
-    if (isCountRowsSet) {
```

Comment:
We should not remove this if as it will break the row count calculation functionality. We need this snippet in place:

```
 if (isCountRowsSet) {
      const footerItemsCountRows: FooterItem[] = [headerGroups[0]?.headers[0]?.filteredRows.length.toString() ?? data.length.toString()];
      setFooterItems(footerItemsCountRows);
      return;
    }
```

Reply:
Good catch! It's back.